pages/integrations/github.md

```diff
@@ -120,6 +120,9 @@ You can customize the commit statuses, for example to reuse the same pipeline fo
 3. When you make a new commit or pull request, you should see _my-custom-status_ as the commit status:
     <%= image "github-custom-status.png", alt: "Screenshot of GitHub build settings and the resulting GitHub pull request statuses" %>
 
+>🚧 Build source limitations when using custom statuses
```

Comment:
Is this specific to custom statuses defined at the build level? Is it also relevant to step level definitions?

Reply:
it's the same for build and step-level definitions